go.mod

```diff
@@ -3,12 +3,16 @@ module github.com/cosmos/gaia
 go 1.14
 
```

Comment:
We (I) screwed up when release v2 of Gaia (the current version on the hub) in that we're not following Go's module semantic version practices [here](https://blog.golang.org/v2-go-modules).

The module must be declared as `module github.com/cosmos/gaia/v3`.

/cc @shahankhatch

Reply:
Good point

Reply:
Cosmos SDK's latest RCs explicitly require `go 1.15`. I suggest to modify line 3 accordingly

Reply:
Yeah we need to decide what to here.

It would be interesting as an approach to have gaia v2 and gaiav3 coexist in the same repo instead of replacing all the v2 code with v3.

Need to make a decision

Reply:
Take https://github.com/go-yaml/yaml as an example. One can have `v2` and `v3` (and `v4`, and so on) living on the same repo.

Reply:
Needless to say, versions must be tagged on their respective _major version_ branches [1]

[1] And I'll never stop recommending using `git tag -a` for *all final releases* (I think I've discussed this with @marbar3778, who agrees with me).

Reply:
If I'm not mistaken, all that is required is to specify `module github.com/cosmos/gaia/v3`.

Reply:
If I do that, that without changing the folder structure, I get `go mod` errors

Reply:
> all that is required is to specify module github.com/cosmos/gaia/v3

Yep (ref: https://golang.org/ref/mod#vcs-dir)

> If I do that, that without changing the folder structure, I get go mod errors

That's weird. Once you changed the root module's name, you should then rename all imports. I'll try take a look at it today.

Reply:
ohhhh maybe we have to change imports too! can you post the error/output @zmanian?

Reply:
Okay I think I got this working in `5652138`